config/locales/es.yml

```diff
@@ -1483,12 +1464,12 @@ es:
           all_copies_html: Tengo <u>todas</u> las copias de mis documentos fiscales oficiales
           does_not_apply_html: "<u>No tengo</u> copias de ningún documento fiscal oficial porque <u>no me aplican</u>"
           need_help_html: "<u>No tengo</u> ninguna copia de mis documentos fiscales oficiales y <u>me gustaría recibir ayuda</u> para obtenerlas."
-          some_copies_html: Tengo <u>algunas</u> copias de mis documentos fiscales oficiales
+          some_copies_html: Tengo <u>algunas</u> copias de mis documentos fiscales oficiales.
         help_text: Los documentos fiscales se refieren a cualquier documento oficial del IRS o documentos que confirmen su empleo como una W2, 1099, registro de trabajo personal, etc.
         title: "¿Qué tipo de documentos fiscales tiene?"
     triage_express:
       edit:
-        title: "¡Le recomendamos que presente su declaración con nuestra opción Express!"
+        title: Le recomendamos que use la opcion rápida
```

Comment:
The translator seems not to be using the brand name "Express" but instead translating the word "Express". That seems wrong to me.